src/components/Header.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <header>
+    <div class="logo">
+      <router-link to="/">Minusz labs</router-link>
```

Comment:
What do you think about directing by name instead of path ?

Reply:
done